### ISSUE TITLE
install from?

### ISSUE THREAD

**Issue by mixmix** [mattfbacon/messagease-better#11]
Do you have pre built installers I can use or do I have to build my own? Thanks

**Comment by mattfbacon** [comment]
No, currently it can be installed with `./gradlew installDebug`

**Comment by mattfbacon** [comment]
Release apks are now built automatically for every commit. Due to the spontaneous and unstructured nature of my development I don't plan to make versioned releases, but hopefully this addresses your problem nonetheless.
